azure-pipelines.yml

```diff
@@ -1,16 +1,23 @@
 variables:
-    { MODULE_VERSION: "2.1.2", NODE_10: "10.x", NODE_12: "12.x", NODE_14: "14.x", NODE_16: "16.x" }
+    {
+        MODULE_VERSION: "2.1.2",
+        NODE_10: "10.x",
+        NODE_12: "12.x",
+        NODE_14: "14.x",
+        NODE_16: "16.x",
+        NODE_18: "18.x",
+    }
 name: $(MODULE_VERSION)-$(Date:yyyyMMdd)$(Rev:.r)
 
 pr:
     branches:
         include:
-            - main
-            - dev
+            - v2.x

```

Comment:
You don't need `v3.x` on this branch

For `pr`:
> The version of the pipeline in the pushed branch is used.

For `trigger`:
> The version of the pipeline in the source branch for the pull request is used.

https://learn.microsoft.com/en-us/azure/devops/pipelines/build/triggers?view=azure-devops#branch-consideration-for-triggers-in-yaml-pipelines

Reply:
Removed 3.x